### PR TITLE
fix a ops.tab file location mistake with OTP source

### PR DESCRIPTION
**Pull request by hitdavid**
file ops.tab is in opt:erts/emulator/beam/emu directory

### CODE REVIEW

**Review by happi — COMMENTED** [review]
That was the sneakiest thing I have seen....

File: FUNDING.yml [hunk 1]
```diff
@@ -1 +1,2 @@
-github: [happihacking]
```
Comment (robertoaloi) [inline FUNDING.yml:1]: O_o

Comment (robertoaloi) [reply FUNDING.yml:1]: Apparently, even if this is rejected, this is how the contributor profile looks like:

Screenshot 2020-10-08 at 09 43 44

But at least it now points to happihacking.

Comment (robertoaloi) [reply FUNDING.yml:1]: It looks like the book is forked for a Chinese translation, which is a nice thing:

https://github.com/hitdavid/theBeamBook

So, let's just assume this was unintentional :)

Comment (hitdavid) [reply FUNDING.yml:1]: OMG
I didn't want to commit the code this way on purpose.
It’s a BIG mistake.
really sorry to author and contributors.

I want to get some infomation about potential Chinese-Version book readers through My sponsorship button, and totally do not intent to change happi’s funding config.
Sorry again.

**Comment by hitdavid** [comment]
> That was the sneakiest thing I have seen....

sorry for the funding.yml thing

**Comment by happi** [comment]
No problem.
Sorry for the late anser I have been very busy.
Please submit a new PR with just the fix.

**Pull request closed without merging**